app/views/farmers/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<nav>
```

Comment:
This will be a note for me to add the farmer and farm links to my garden and plant pages